dev/archery/archery/crossbow/core.py

```diff
@@ -536,17 +536,34 @@ def _latest_prefix_id(self, prefix):
             latest = -1
         return latest
 
+    def _latest_prefix_date(self, prefix):
+        pattern = re.compile(r'[\w\/-]*{}-(\d+)-(\d+)-(\d+)'.format(prefix))
+        matches = list(filter(None, map(pattern.match, self.repo.branches)))
+        if matches:
+            latest = sorted([m.group(0) for m in matches])[-1][-10:]
```

Comment:
I am not sure what the `[-10:]` pertains to, so maybe add a comment. Or could this be extracted using regex for more generality?

Reply:
Added a comment. 
(I don't think a regex is needed, since the match itself is already done with a regex, which should ensure that it ends with this date. Of course in theory the date could change format, but if we use ISO date, it will always be like this)